sources/zafira-dbaccess/src/main/resources/db/changelog.yml

```diff
@@ -66,13 +66,24 @@ databaseChangeLog:
             relativeToChangelogFile: true
             splitStatements: false
   - changeSet:
-        id: 007_drop_tests_test_runs_fk
-        validCheckSum: ANY
-        labels: stable
-        author: nsidorevich
-        changes:
-            - sqlFile:
-                  encoding: utf8
-                  path: changeset/007_drop_tests_test_runs_fk.sql
-                  relativeToChangelogFile: true
-                  splitStatements: false
+      id: 007_drop_tests_test_runs_fk
+      validCheckSum: ANY
+      labels: stable
+      author: nsidorevich
+      changes:
+        - sqlFile:
+            encoding: utf8
+            path: changeset/007_drop_tests_test_runs_fk.sql
+            relativeToChangelogFile: true
+            splitStatements: false
+  - changeSet:
+      id: 008_drop_message_broker_integration
+      validCheckSum: ANY
+      labels: stable
+      author: brutskov
+      changes:
+          - sqlFile:
+              encoding: utf8
+              path: changeset/008_drop_message_broker_integration.sql
```

Comment:
Check please, that zafira is able to start  based on this config, I'm not sure that formatting is correct.